hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+Copyright YEAR The Kubernetes Authors All rights reserved.
```

Comment:
@hferentschik The easiest way to skip the files with Kubernetes boilerplate .But it is not right in my opinion as we might get new files from minikube in future. The current implementations is very deeply tied up with one boilerplate for one type of file.  I am still figuring out what would be the right thing to do here.